Documentation/Basics.md

```diff
@@ -1,66 +1,57 @@
-# Basic Usage #
-
-Let's say you have an endpoint, `GET https://www.example.com/comments/1`, that returns the following JSON:
+Let's say we have an endpoint, `https://raw.githubusercontent.com/thoughtbot/Swish/master/Documentation/example.json`
+that returns the following JSON:
 
 ```json
 {
-  "id": 1,
+  "id": 4,
   "commentText": "Pretty good. Pret-ty pre-ty pre-ty good.",
   "username": "LarryDavid"
 }
 ```
 
-We'll model this with the following struct, and implement Argo's `Decodable` protocol to tell it how to deal with JSON:
+We will model the data with the following struct,
+and implement the `Codable` protocol so we can
+tell it how to turn from JSON into a Swift object.
 
 ```swift
-import Argo
-import Curry
-
-struct Comment {
-  let id: Int
-  let text: String
-  let username: String
-}
-
-extension Comment: Decodable {
-  static func decode(_ json: JSON) -> Decoded<Comment> {
-    return curry(Comment.init)
-      <^> j <| "id"
-      <*> j <| "commentText"
-      <*> j <| "username"
-  }
+struct Comment: Codable {
+    let id: Int
+    let body: String
+    let username: String
+
+    enum CodingKeys: String, CodingKey {
+        case id = "id"
+        case body = "commentText"
+        case username = "username"
+    }
 }
 ```
 
-We can model the request by defining a struct that implement's Swish's `Request` protocol:
+We can model the request by defining a struct that
+implements Swish's `Request` protocol
 
 ```swift
-import Swish
-
 struct CommentRequest: Request {
-  typealias ResponseObject = Comment
+    typealias ResponseObject = Comment
 
-  let id: Int
 
-  func build() -> URLRequest {
-    let url = URL(string: "https://www.example.com/comments/\(id)")!
-    return URLRequest(URL: url)
-  }
+    func build() -> URLRequest {
+        let url = URL(string: "https://raw.githubusercontent.com/thoughtbot/Swish/master/Documentation/example.json")!
+        return URLRequest(url: url)
+    }
 }
 ```
 
-We can then use the Swish's default `APIClient` to make the request:
+We can then use the Swish's default APIClient to make the request:
```

Comment:
Do you think it makes sense to leave the backticks in here?

Reply:
Yep!